packages/packages/commonwell-cert-runner/src/link-management.ts

```diff
@@ -18,7 +18,7 @@ export async function linkManagement(commonWell: CommonWell, queryMeta: RequestM
   );
   const patientId = getIdTrailingSlash(respPatient);
   const referenceLink = respPatient._links.self.href;
-  const respC5a = await commonWell.addPatientLink(queryMeta, personId, referenceLink);
+  const respC5a = await commonWell.patientLink(queryMeta, personId, referenceLink);
```

Comment:
we deprecated `patientLink`, using `addPatientLink` now

here and below (multiple instances of this comment)